src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -470,6 +470,11 @@ In this message, the term "binding" means a piece of generated code that makes i
     <value>The 'RunAOTCompilation' MSBuild property is only supported when trimming is enabled. Edit the project file in a text editor to set 'PublishTrimmed' to 'true' for this build configuration.</value>
     <comment>The following are literal names and should not be translated: 'RunAOTCompilation', 'PublishTrimmed'</comment>
   </data>
+  <data name="XA1031" xml:space="preserve">
+    <value>The 'AndroidHttpClientHandlerType' has an invalid value of '{0}' please check your project settings.</value>
```

Comment:
Maybe this one should say something like:
```suggestion
    <value>The 'AndroidHttpClientHandlerType' has an invalid value of '{0}' please change the value in your project file to one of X, Y, or remove the property completely.</value>
```

Reply:
Error messages should be actionable.  *How* is the value invalid?

Suggested alternative:

```
The 'AndroidHttpClientHandlerType' property value '{0}' must derive from '{1}'.
Please change the value to an assembly-qualifed type name which inherits from '{1}' or remove the property completely.
```